Sources/fs/Directory.swift

```diff
@@ -9,34 +9,30 @@ import Dispatch
 import xsys
 import core
 
-public func readdir(_ path: String, cb: @escaping ( [ String ]? ) -> Void) {
+public func readdir(_ path: String, cb: @escaping ( Error?, [ String ]? ) -> Void) {
```

Comment:
I don't want to be picky, but I'd prefer if pull-requests would do just a single thing. Hiding API changes like that in a Swift-3.1-warning-fix pull is a little inconvenient ;->